generate_env.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Define the content of the .env file
+env_content="ENV=local\nKEY_VAULT-NAME=ti-key-vault-name\nREPORT_STREAM_URL_PREFIX=http://localhost:7071"
```

Comment:
Do we need a .env.example as well as a script to generate it?

Reply:
Good catch. The .env.example file was made before the script. It is no longer needed.